nixio/test/test_doc_examples.py

```diff
@@ -43,34 +42,40 @@ def setUp(self):
             self.examples_path = Path.joinpath(Path(curr_path).parent.parent,
                                                "docs", "source", "examples")
 
+        util_path = Path.joinpath(self.examples_path, "docutils.py")
+        spec = importlib.util.spec_from_file_location("docutils", util_path)
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+        sys.modules["docutils"] = module
+
         # Render matplotlib plots non-blocking
         plt.ion()
 
     def tearDown(self):
         plt.close("all")
         plt.ioff()
         if Path.exists(Path("TEST_IMAGE")):
-            Path.unlink(Path(TEST_IMAGE))
+            Path.unlink(Path(TEST_IMAGE), True)
```

Comment:
Unrelated to this PR but should the line above be `Path.exists(Path(TEST_IMAGE))` (instead of the string `"TEST_IMAGE"`)?

Reply:
good catch!